Actions/AL-Go-Helper.ps1

```diff
@@ -652,6 +652,82 @@ function ExcludeUnneededApps {
     }
 }
 
+function ResolveProjectFolders {
+    Param(
+        [string] $project,
+        [string] $baseFolder,
+        [ref] $projectSettings
+    )
+    $projectPath = Join-Path $baseFolder $project -Resolve
+
+    Push-Location $projectPath
+
+    try {
+        # Filter out app folders that doesn't contain an app.json file
+        'appFolders', 'testFolders', 'bcptTestFolders' | ForEach-Object {
+            $folders = $projectSettings.Value.$_
+            
+            $resolvedPaths = @()
+            foreach($folder in $folders) {
+                $aLProjectFolder = Join-Path $projectPath $folder
+                $resolvedALProjectsPaths = Resolve-Path $aLProjectFolder -ErrorAction Ignore | Where-Object { Test-Path (Join-Path $_ 'app.json') }
+
+                if($resolvedALProjectsPaths) {
+                    $resolvedPaths += @($resolvedALProjectsPaths)
+                }
+                else {
+                    OutputWarning "The folder '$aLProjectFolder' cannot be resolved or does not contain an app.json file. Skipping."
```

Comment:
Maybe: The folder '$folder' in project '$project' cannot be.... - in order to not show the full path of the agent